src/controllers/queries/index.ts

```diff
@@ -406,6 +406,7 @@ export class QueriesController extends HTMLController {
         },
       })
     } catch (err) {
+      log.warn('error occured while submitting a query request', err?.toString())
```

Comment:
To avoid using `"as unknown as ..."` and due to JS magic it would render undefined if err isn't undefined, to be fair in TS error is unknown since it can be Error or Exception and maybe more not 100% sure here. I could do something like
```
} catch (error) {
	if (error instanceof Error) return log.warn(error.message)
	log.warn('unknown error')
```

Reply:
the only reason I'm asking is cuz I saw somewhere that doing `err.toString()` gets rid of the stack trace

Reply:
yes it does, this is also going to be updated with the below one